tests/ut_common_copytotarget.py

```diff
@@ -0,0 +1,27 @@
+"""
+The unit test check if copy_to_target(file_expr, target) works fine.
+
+"""
+import sys
+import os
+import shutil
+import tempfile
+import build_component
+
+def main():
+    # create a new temporary file and a new temporary directory.
+    temp = tempfile.NamedTemporaryFile()
+    tempdir = tempfile.mkdtemp()
+
+    # copy a temporary file to a temporary directory.
+    build_component.copy_to_target(temp.name, tempdir)
+    
+    # check if this funtion works fine.
+    if not os.path.isfile(tempdir+ os.path.sep + os.path.basename(temp.name)):
+        print "this function is failed"
```

Comment:
It would be good to give an explanation, just in case someone runs this unit test outside of `utf.py`.

Suggestion: "build_component.copy_to_target failed to copy file ... to folder ...."